### PR TITLE
[14.0] [MIG] l10n_us_form_1099

### PR DESCRIPTION
**Pull request by JevinD**
l10n_us_form_1099 migration from 13.0 to 14.0

### CODE REVIEW

**Comment by JevinD** [comment]
@bodedra  Can you review this and if completed properly can you ping @max3903 for final review.

**Review by bodedra — APPROVED** [review]
Functional reviewed. :+1: 

Minor suggestion:

s/ Go to Accounting > Purchases / Go to Invoicing > Bills > Vendors

s/Go to Accounting > Reporting > 1099 Report / Go to Invoicing > Reporting > 1099 Report

**Comment by JevinD** [comment]
> Functional reviewed. 👍
> 
> Minor suggestion:
> 
> s/ Go to Accounting > Purchases / Go to Invoicing > Bills > Vendors
> 
> s/Go to Accounting > Reporting > 1099 Report / Go to Invoicing > Reporting > 1099 Report

Minor Suggestions have been made.

**Review by bodedra — APPROVED** [review]

**Review by brian10048 — APPROVED** [review]
Functional test - Everything looks good

**Comment by dreispt** [comment]
@JevinD Mr. Travis is not happy: https://travis-ci.com/github/OCA/l10n-usa/jobs/454540985#L408

File: l10n_us_form_1099/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "data/type_1099_data.xml",
+        "data/box_1099_misc_data.xml",
+        "security/ir.model.access.csv",
+        "views/type_1099_view.xml",
+        "views/box_1099_misc_view.xml",
+        "views/res_partner.xml",
+        "reports/account_payment_1099_report_views.xml",
+    ],
+    "installable": True,
+    "development_status": "Stable",
```
Comment (bodedra) [inline l10n_us_form_1099/__manifest__.py]:
```suggestion
    "development_status": "Production/Stable",
```

**Comment by JevinD** [comment]
@dreispt @bodedra Travis checks are green

**Pull request merged**